elixir/apps/web/lib/web/mailer/auth_email/sign_in_link.html.heex

```diff
@@ -1,41 +1,212 @@
-<h3>Sign in token</h3>
```

Comment:
I think to do it properly we also need to include `doctype`, `html`, `body` and at least `meta content-type` tags. HTML by spec should be valid HTML page.

See https://www.campaignmonitor.com/blog/email-marketing/correct-doctype-to-use-in-html-email/

You can also check compatibility of our HTML here: https://vmail.leopard.in.ua/ (great job @le0pard!)

Reply:
Ah, ok, that's no problem.  I made an incorrect assumption that the `doctype`, `html`, `body`, etc... tags were being set elsewhere in the app and this file was just the inner portion of the body that needed to be set.  I can get all of this added.

Reply:
Yeah, I did not add them because emails were not styled at all but since we added styling I guess the time has come to worry about compatibility

Reply:
👍 Understandable.
Just added the entire html doc.